Dockerfile

```diff
@@ -22,10 +20,12 @@ ADD requirements.txt /code/requirements.txt
 ADD package.json /code/package.json
 ADD app/assets /code/app/assets
 
-RUN pip3 install -U -I -r /code/requirements.txt
+RUN git clone git://github.com/yyuu/pyenv.git .pyenv
+ENV PYENV_ROOT /code/.pyenv
+ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
 
 ADD . /code
 
-RUN yarn compile
+RUN pyenv install &&  pyenv rehash && pip install -U pip setuptools wheel
 
-ENTRYPOINT python3 application.py runserver
+ENTRYPOINT pip install -r requirements.txt && yarn compile && python application.py runserver
```

Comment:
so that you don't need to re-build the whole container whenever requirements.txt or yarn compile is needed; restarting the container will do these if it's part of the entry point; it's fast as it only does something if there are changes so maybe adds 5 seconds to the startup time vs. 15 minutes to do a container rebuild